README.md

```diff
@@ -1,2 +1,149 @@
-# leaderboard
-This project aims to make use of gitflow, async functions, webpack and hitting of API's
+# Leaderboard
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [👥 Collaborator](#collaborator)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Loeaderboard] <a name="about-project"></a>
+
+**[Leaderboard]** is a project that aims to integrate promises and callback and async functions in the calling in the hitting of an api.
+
+<br>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Linters</a></li>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[CSS]**
+- **[JavaScript]**
+- **[Linters]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project, you will need the following:
+
+- Web Browser
+- Code Editor
+- Git
+
+### Setup
+
+To get started, clone this repository to your desired folder:
+
+```shell
+cd my-folder
+git clone https://github.com/Samuelwanza/todolist/.git
```

Comment:
- [ ] You are still making use of the repo for your todolist project. please kindly change this to leaderboard so users are directly to the correct repo :+1: